data/templates/unattend_server2012.xml

```diff
@@ -109,12 +109,18 @@
                 </SynchronousCommand>
                 <SynchronousCommand wcm:action="add">
                     <Order>2</Order>
+                    <CommandLine>powershell -Command "Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False" </CommandLine>
+                    <Description>disable firewall on node to be able to ping the new IP when max payload is used</Description>
+                    <RequiresUserInput>false</RequiresUserInput>

```

Comment:
I have some security concern for disable the firewall forever. I think the disable of firewall is to bypass some problem so that the install windows can work well. For a short while, I think it is OK, but I hope it can be enabled finally.